### PR TITLE
update deps to phpunit 6.x, fix code and tests

### LINKED ISSUE #96: PHPUnit 6 Support
Is there currently a plan in place to update the PHPUnit assertions to use the new namespaces in PHPUnit 6?

### PR DESCRIPTION
**Pull request by andreasschacht**
fix #96

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,22 +8,16 @@ cache:
...
+    - php: 7.1
     - php: hhvm
 
 before_install:
   - composer self-update
 
 install:
   - export COMPOSER_ROOT_VERSION=dev-master
-  - if [ "$DEPENDENCIES" != "low" ]; then composer update; fi;
-  - if [ "$DEPENDENCIES" == "low" ]; then composer update --prefer-lowest; fi;
```
Comment (norzechowicz) [inline .travis.yml]: I don't like removing low dependencies tests, please bring back `DEPENDENCIES='low'` for php 7.0

File: .travis.yml [hunk 2]
```diff
@@ -8,22 +8,16 @@ cache:
...
   include:
-    - php: 5.3
-      env: DEPENDENCIES='low'
-    - php: 5.3
-    - php: 5.4
-    - php: 5.5
-    - php: 5.6
     - php: 7.0
+    - php: 7.1
     - php: hhvm
```
Comment (norzechowicz) [inline .travis.yml]: I believe you can remove hhvm as well

File: .travis.yml (hunk 1 above)
Comment (andreasschacht) [inline .travis.yml]: okay, thought it was depending on php version 5.3...

**Comment by norzechowicz** [comment]
overall this PR looks good but before I merge it I'll first stabilize version `2.1` so we can user master as next `3.0`

File: .travis.yml (hunk 2 above)
Comment (andreasschacht) [inline .travis.yml]: fine, recently saw that travis support for hhvm failed in this configuration...

File: .travis.yml (hunk 1 above)
Comment (norzechowicz) [inline .travis.yml]: `low` means that we are testing lowest possible versions of all dependencies

**Pull request merged**